engine/tester/config_parse.c

```diff
@@ -536,6 +650,67 @@ get_persons_info(xmlNodePtr *node, const char *node_name,
     return 0;
 }
 
+/**
+ * Get syntax flags.
+ *
+ * @param node      Location of the XML node pointer.
+ * @param flags     Syntax flags.
+ *
+ * @return Status code.
+ */
+static te_errno
+get_syntax_flags(xmlNodePtr *node, tester_flags *flags)
+{
+    char       *val;
+    te_errno    rc;
+    xmlNodePtr  child_node;
+
+    assert(*node != NULL);
+    assert(flags != NULL);
+
+    /* 'syntax' block is optional. */
+    if (xmlStrcmp((*node)->name, CONST_CHAR2XML("syntax")) != 0)
+        return 0;
+
+    child_node = xmlNodeChildren(*node);
+    *node = xmlNodeNext(*node);
+    if (child_node == NULL)
+    {
+        ERROR("Empty syntax section");
+        return TE_RC(TE_TESTER, TE_EINVAL);
+    }
+
+    if (xmlStrcmp(child_node->name, CONST_CHAR2XML("markdown")) == 0)
+    {
+        if ((rc = get_node_with_text_content(&child_node, "markdown",
```

Comment:
I am a bit confused by the use of `markdown` keyword here. If only it were documented in the schema...

Reply:
I've added description in Tester XML schema.